MCprep_addon/world_tools.py

```diff
@@ -310,8 +310,34 @@ def execute(self, context):
 		obj_import_mem_msg = (
 			"Memory error during OBJ import, try exporting a smaller world")
 		try:
-			res = bpy.ops.import_scene.obj(
-				filepath=self.filepath, use_split_groups=True)
+			BLENDER_STANDARD = (
+				"Standard",
+				"Filmic",
+				"Filmic Log",
+				"Raw",
+				"False Color"
+			)
+			MTL = self.filepath.rsplit(".", 1)[0] + '.mtl'
+			LINES = None
+			if bpy.context.scene.view_settings.view_transform not in BLENDER_STANDARD:
+				with open(MTL, 'r') as mtl_file:
+					LINES = mtl_file.readlines()
+					for index, line in enumerate(LINES):
+						if line.startswith("map_d"):
+							LINES[index] = "# " + line
+				with open(MTL, 'w') as mtl_file:
+					mtl_file.writelines(LINES)
+
+			res = None
+			if util.min_bv((3, 5)):
+				res = bpy.ops.wm.obj_import(
+					filepath=self.filepath, use_split_groups=True) # Returns functionality missing in 3.1 - 3.4 
+			if util.min_bv((3, 1)):
```

Comment:
```suggestion
			elif util.min_bv((3, 1)):
```

Otherwise wouldn't it import twice?